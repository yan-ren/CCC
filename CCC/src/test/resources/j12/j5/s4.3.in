4
4 3 2 1
5
5 4 3 2 1
5
4 3 2 5 1
5
2 4 5 1 3
0
