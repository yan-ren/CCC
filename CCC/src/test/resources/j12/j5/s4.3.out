22
32
20
16
